user_guide_src/source/installation/running.rst

```diff
@@ -25,31 +25,53 @@ section of the User Guide to begin learning how to build dynamic PHP application
 Initial Configuration
 *********************
 
-#. Open the **app/Config/App.php** file with a text editor and
-   set your base URL to ``$baseURL``. If you need more flexibility, the baseURL may
-   be set within the :ref:`.env <dotenv-file>` file as ``app.baseURL = 'http://example.com/'``.
-   (Always use a trailing slash on your base URL!)
+Configure for Your Site URIs
+============================
+
+Open the **app/Config/App.php** file with a text editor.
+
+#. $baseURL
+    Set your base URL to ``$baseURL``. If you need more flexibility, the baseURL may
+    be set within the :ref:`.env <dotenv-file>` file as ``app.baseURL = 'http://example.com/'``.
+    (Always use a trailing slash on your base URL!)
```

Comment:
I saw some people having problems in the forum. I think it is prominently bold.

Reply:
You mean to make "Always use a trailing slash on your base URL!" bold?

Reply:
Sorry. Yes.

Reply:
Done!